Makefile

```diff
@@ -327,6 +327,9 @@ $(foreach ITEM,$(PILOT_GO_BINS_SHORT),$(eval $(call pilotbuild,$(ITEM))))
 istioctl ${ISTIO_OUT}/istioctl:
 	bin/gobuild.sh ${ISTIO_OUT}/istioctl ./istioctl/cmd/istioctl
 
+istioctl-gzip:
+    gzip -9 ${ISTIO_OUT}/istioctl 
```

Comment:
We already have the release gziped, now they have to ungzip the release and then ungzip istioctl? Is that right? If so this seems bad for UX, and not any benefit - gzip twice doesn't help afaik.

Have you compared the size before and after?

Reply:
@howardjohn yes we have. take a look at what @esnible did in #13064- gzip with -9 takes the istioctl binary down to 12.8 MB. This PR is an effort to make the istioctl footprint smaller, but I understand what you mean with respect to UX (gunzipping twice isn't exactly the best UX). Now that this PR has opened up a discussion on this, what could we do to make the istioctl binary smaller, w/o requiring users to gunzip twice?

Reply:
We already gzip the release though... 1.1.4 is only 15 MB which is roughly the same as what Ed got. If the `-9` really adds a lot then just add that flag to the gzip of the release and call it a day. Gziping istioctl and then gziping the release (which has istioctl) isn't going to do anything except kill UX.

Reply:
Understood. I'll take a look at doing that instead. @esnible thoughts on this?

Reply:
Howard John is right.  What I propose is adding `-s -w` to _gobuild.sh_ (but perhaps just for _istioctl_?)  I propose adding `-9` to the zipping of the release.